src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -26,31 +26,58 @@ export interface Props extends ContextProps {
   user: User
 }
 
+// TODO: This whole component should be refactored to use less `Media` decisions
```

Comment:
These components are slightly too complicated to do immediately, I wanted to get the layout fixed first.